src/serializers/resource.js

```diff
@@ -141,7 +158,35 @@ function ResourceSerializer(Implementation, model, records, integrator, opts, me
       });
     }
 
-    var serializationOptions = {
+    function defineRelationshipId(records, fields, goDeeper = true) {
```

Comment:
Recursive function to initialize belongsTo with an id if not set (based on foreignKey) so the serializer can return the `{ data: { id: 2 } }`